### PR TITLE
docs(spec v2): fixes to 2 discrepancies between this specs and Pathfinder Framework

### PR DESCRIPTION
**Pull request by zeitgeist**
(see the tech spec's changelog section for additional details)

### CODE REVIEW

**Comment by marleyg** [comment]
Fixed a typo in the description for referencePeriodEnd.

File: spec/v2/index.bs [hunk 1]
```diff
@@ -2169,6 +2183,26 @@ path: LICENSE.md
...
 # Appendix B: Changelog # {#changelog}
 
+
+## Version 2.0.1-20230301 (Mar 1, 2023) ## {#changelog-2.0.1-20230301}
+
+This version fixes 2 discrepancies between the Pathfinder Framework 
+Version 2 and the data model in this specification.
+
+1. property <{CarbonFootprint/boundaryProcessesDescription}>: was defined as
+    optional is now mandatory in accordance with Pathfinder Framework Version 2
```
Comment (bhadley) [inline spec/v2/index.bs:2193]: to clarify, I would write: "was incorrectly defined as optional in v2.0.0, and this typo is now corrected such that the property is correctly marked as mandatory in accordance with the Pathfinder Framework Version 2"

**Review by bhadley — APPROVED** [review]
Looks good to me  - just made two comments in the Version notes

File: spec/v2/index.bs [hunk 2]
```diff
@@ -2169,6 +2183,26 @@ path: LICENSE.md
...
+2. update to definition of property <{CarbonFootprint/primaryDataShare}>: 
+    it was marked as optional (`O`) and is now marked as `O*`. This update is 
+    accordance with the Pathfinder Framework and the field's previous (v2.0.0) semantics;
```
[the remark this answers is not in the document]
Comment (zeitgeist) [reply spec/v2/index.bs]: fixed with f6172c1a9777d9f10471b030ddba2ad2be0bb0f7

**Pull request merged**